modules/lang-mustache/src/main/java/org/elasticsearch/script/mustache/CustomMustacheFactory.java

```diff
@@ -52,7 +52,7 @@
 
 public class CustomMustacheFactory extends DefaultMustacheFactory {
 
-    static final String JSON_MIME_TYPE_WITH_CHARSET = "application/json; charset=UTF-8";
+    static final String JSON_MIME_TYPE_WITH_CHARSET = "application/json;charset=utf-8";
```

Comment:
I replaced this one to be consistent with what ES will now return. 
no space after media type
and lowercased parameters

Reply:
+1 on consistency ... just want to make sure it is not _required_ since these should be case insensitive right ?

Reply:
in fact.. this is a great question because it might not be.. 
https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/script/Script.java#L636
here a simple equals is used on the option provided vs xcontentType.mediaType (I assume it can be provided by a user, need to confirm).
I will revert the changes around scripting and raise an issue to discuss with scripting team